src/library/Uno.Material/Controls/Chips/ChipGroup.cs

```diff
@@ -27,6 +27,8 @@ public partial class ChipGroup : ItemsControl
 
 		private bool _isSynchronizingSelection = false;
 		private bool _isUpdatingSelection = false;
+		private bool _needsToSynchronizeInitialSelectionForSelectedItem = false;
```

Comment:
You could combine that into one variable, since `SelectedItem` and `-Items` shouldnt be both used. Plus, at `Loaded`, you can be sure that `SelectionMode` is set, and based on that you can decide which take priority.